### ISSUE TITLE
QUESTION

### ISSUE THREAD

**Issue by TimBim-jpg** [Tschipcraft/spawnanimations#12]
### Is your suggested enhancement related to a problem? Please describe.

Is there a way to make so ONLY little poof particles appear, without digging out animation, and also i would like this animation on every mob, even friendly(not necessary), or if not, please suggest some alternatives

**Comment by Tschipcraft** [comment]
Yes, you would need to add all mobs that you want to see the poof animation on to **both** (yes, doesn't make much sense, but that's how it works rn) `dig_up_animation.json` and `always_poof_animation.json` located in _/data/spawnanimations/tags/entity_types/_.

**Comment by TimBim-jpg** [comment]
> 

thanks, will check that out

**Comment by TimBim-jpg** [comment]
> 
oh wait but where do i find that file path if its a mod?, in my minecraft data folder i dont have such subfolders either

**Comment by Tschipcraft** [comment]
Ah, well you would have to unzip the mod first with a third party app like 7-zip.
I'd recommend sticking with the data pack version for now, if you don't want to install additional software. After modifying the files, you can re-zip the data pack.

I'm currently working on building the mod version myself for the future and not relying on Modrinth, so when I'm done you could just easily clone the repository, change the files, zip everything with file explorer and just change the extension to .jar again.

**Comment by Tschipcraft** [comment]
Updated the wiki guide to include instructions for the mod version, closing this issue now
